Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,588 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import urllib
+from typing import List, Dict, Tuple
+import pandas as pd
+import base64
+import requests
+import dill
+import copy
+import numpy as np
+from tldextract import TLDExtract
+requests.packages.urllib3.disable_warnings()
+
+dill.settings['recurse'] = True
+
+no_fetch_extract = TLDExtract(suffix_list_urls=None, cache_dir=False)
+
+OOB_MAJOR_VERSION_INFO_KEY = 'major'
+OOB_MINOR_VERSION_INFO_KEY = 'minor'
+MAJOR_VERSION = 0
+MINOR_DEFAULT_VERSION = 0
+
+MSG_MODEL_VERSION_IN_DEMISTO = "Model version in demisto: %s.%s"
+MSG_NO_MODEL_IN_DEMISTO = "There is no existing model version in demisto"
+MSG_INVALID_URL = "Error: "
+MSG_NO_URL_GIVEN = "Please input at least one URL"
+MSG_FAILED_RASTERIZE = "Rasterize for this url did not work correctly"
+MSG_IMPOSSIBLE_CONNECTION = "Failed to establish a new connection - Name or service not known"
+MSG_UPDATE_MODEL = "Update demisto model from docker model version %s.%s"
+MSG_UPDATE_LOGO = "Update demisto model from docker model version %s.%s and transfering logos from demisto version %s.%s"
+MSG_WRONG_CONFIG_MODEL = 'Wrong configuration of the model'
+MSG_NO_ACTION_ON_MODEL = "Use current model"
+MSG_WHITE_LIST = "White List"
+EMPTY_STRING = ""
+URL_PHISHING_MODEL_NAME = "url_phishing_model"
+OUT_OF_THE_BOX_MODEL_PATH = '/model/model_docker.pkl'
+UNKNOWN_MODEL_TYPE = 'UNKNOWN_MODEL_TYPE'
+THRESHOLD_NEW_DOMAIN_MONTHS = 6
+DOMAIN_AGE_KEY = 'New domain (less than %s months)' % str(THRESHOLD_NEW_DOMAIN_MONTHS)
+
+MALICIOUS_VERDICT = "Malicious"
+BENIGN_VERDICT = "Benign"
+SUSPICIOUS_VERDICT = "Suspicious"
+BENIGN_VERDICT_WHITELIST = "Benign - whitelisted"
+
+BENIGN_THRESHOLD = 0.5
+SUSPICIOUS_THRESHOLD = 0.7
+
+SCORE_INVALID_URL = -1.0
+SCORE_BENIGN = 0.0  # type: float
+
+GREEN_COLOR = "{{color:#1DB846}}(%s)"
+RED_COLOR = "{{color:#D13C3C}}(%s)"
+
+VERDICT_MALICIOUS_COLOR = "{{color:#D13C3C}}(**%s**)"
+VERDICT_SUSPICIOUS_COLOR = "{{color:#EF9700}}(**%s**)"
+VERDICT_BENIGN_COLOR = "{{color:#1DB846}}(**%s**)"
+VERDICT_ERROR_COLOR = "{{color:#D13C3C}}(**%s**)"
+MAPPING_VERDICT_COLOR = {MALICIOUS_VERDICT: VERDICT_MALICIOUS_COLOR, BENIGN_VERDICT: VERDICT_BENIGN_COLOR,
+                         SUSPICIOUS_VERDICT: VERDICT_SUSPICIOUS_COLOR, BENIGN_VERDICT_WHITELIST: VERDICT_BENIGN_COLOR}
+
+SCORE_THRESHOLD = 0.7  # type: float
+
+STATUS_CODE_VALID = 200
+
+MODEL_KEY_URL_SCORE = 'url_score'
+MODEL_KEY_LOGO_FOUND = 'logo_found'
+MODEL_KEY_SEO = 'seo'
+MODEL_KEY_LOGO_IMAGE_BYTES = 'image_bytes'
+MODEL_KEY_LOGIN_FORM = 'login_form'
+
+KEY_CONTENT_DOMAIN = "Domain"
+KEY_CONTENT_URL = "URL"
+KEY_CONTENT_LOGO = "SuspiciousLogo"
+KEY_CONTENT_LOGIN = "LoginForm"
+KEY_CONTENT_URL_SCORE = "URLScore"
+KEY_CONTENT_SEO = "ContentBasedVerdict"
+KEY_CONTENT_AGE = "DomainAge"
+
+KEY_FINAL_VERDICT = "Final Verdict"
```

Comment:
Remove space please from value so it won't cause issues

Reply:
No it's for HR not context here

Reply:
![image](https://user-images.githubusercontent.com/57318201/135443767-12ffe4f3-5087-4ec0-b01f-24d7104d32f2.png)